src/ui/components/Toolbar.tsx

```diff
@@ -423,14 +423,16 @@ export default function Toolbar() {
         {testRunner !== "cypress" && showTour ? (
           <ToolbarButton icon="tour" name="tour" label="Replay Tour" onClick={handleButtonClick} />
         ) : null}
-        {showPassport ? (
+
+        {showPassport && testRunner === null ? (
```

Comment:
nit: `showPassport` as a name for a variable doesn't make as much sense if it's not the only thing we consider before actually showing the passport.

Maybe we could name this `showPassportPreference` or something?